rules/test/lldb/lldb_sim_runner.py

```diff
@@ -8,23 +8,21 @@
 import time
 import threading
 import traceback
-import tempfile
-import shutil
-import json
 import contextlib
 
 logging.basicConfig(
     format="%(asctime)s.%(msecs)03d %(levelname)s %(message)s",
     datefmt="%Y-%m-%d %H:%M:%S",
-    level=logging.INFO)
```

Comment:
The formatting changes are adding a lot of extra stuff onto the Xcode bump PR - I'd suggest to break something like this out into a smaller one in general

Reply:
Yeah, I can revert this for now.

We have `buildifier` to lint/format all the starlark, which is the bulk of the repo, but the situation with bash, python, and whatever else we have floating around is pretty inconsistent. I was removing unused imports here, and `black` aggressively tried to take care of inconsistent quotes, dangling whitespace, etc.